scripts/install_third_party_dependencies.sh

```diff
@@ -1,49 +1,20 @@
 #!/bin/bash
-CONDA_INSTALL_URL=${CONDA_INSTALL_URL:-"https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh"}
-
-source scripts/vars.sh
-
-# Install Miniconda locally
-rm -rf lib/conda
-rm -f /tmp/Miniconda3-latest-Linux-x86_64.sh
-wget -P /tmp \
-    "${CONDA_INSTALL_URL}" \
-    && bash /tmp/Miniconda3-latest-Linux-x86_64.sh -b -p lib/conda \
-    && rm /tmp/Miniconda3-latest-Linux-x86_64.sh
-
-# Grab conda-only packages
-export PATH=lib/conda/bin:$PATH
-lib/conda/bin/python3 -m pip install nvidia-pyindex
```

Comment:
What was this nvidia pyindex for actually?

Reply:
I'm not exactly sure. According to https://github.com/nvidia/tensorflow/#install it's supposed to host NVIDIA wheels that are not part of the regular pypi package, but I didn't find anything that requires it? Especially now that we install flash attention using pip rather than building manually from source?

Maybe @gahdritz would know more.

Reply:
With the current environment.yml, I don't this is used anymore, it seems like it was a specific channel for pulling pip packages.